src/Sfa.Roatp.Register.Infrastructure/Elasticsearch/ProviderRepository.cs

```diff
@@ -69,7 +69,7 @@ public ProviderDocument GetProvider(int ukprn)
             if (results.ApiCall.HttpStatusCode != 200)
             {
                 _log.Warn($"httpStatusCode was {results.ApiCall.HttpStatusCode}");
-                throw new ApplicationException("Failed query all providers");
+                throw new ApplicationException($"Failed query provider with ukprn: {ukprn}");
```

Comment:
This line looks like a test should be updated.  Is there a test to update?  I'm not holding up approval on this issue, but have a look and update the test to be more specific maybe?